src/main/kotlin/com/dmdirc/App.kt

```diff
@@ -49,7 +49,17 @@ class MainApp : Application() {
 private fun createKodein(stage: Stage, hostServices: HostServices, titleProperty: StringProperty) = Kodein {
     bind<String>("version") with singleton { getVersion() }
     bind<Path>() with singleton { getConfigDirectory() }
-    bind<ClientConfig>() with singleton { ClientConfig.loadFrom(instance<Path>().resolve("config.yml")) }
+    bind<ClientConfig>() with singleton {
```

Comment:
Wouldn't it make more sense to do this in the bind<Path> above, so it applies to everything we load?